scripts/asbcli/asbcli

```diff
@@ -460,14 +468,16 @@ class App(object):
 # 24   -v \
 # 25   http://cap.example.com:1338/v2/service_instances/$instanceUUID
 ############################################################
-    def _req_provision(self, service_id, answers):
+    def _req_provision(self, service_id, answers, namespace):
```

Comment:
bind, unbind, and deprovision get their namespaces from the service instance that the broker is caching: `instance.Context`.  There is no support for passing in namespace for bind, unbind,  & deprovision. Also the openservicebroker api does not define a context for those methods.